### PR TITLE
[SP login] use new sp login ui

### PR DESCRIPTION
**Pull request by andxu**

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/ui/SignInWindow.java [hunk 1]
```diff
@@ -188,22 +164,21 @@ protected JComponent createCenterPanel() {
         return AzureTaskManager.getInstance().runInModalAsObservable(task).toSingle();
     }
 
+    private static final IdeaSecureStore secureStore = IdeaSecureStore.getInstance();
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/ui/SignInWindow.java:167]: move to header

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/ui/SignInWindow.java [hunk 2]
```diff
@@ -214,6 +189,18 @@ protected JComponent createCenterPanel() {
         return authMethodDetailsResult;
     }
 
+    private void doServicePrincipalLogin() {
+        final ServicePrincipalLoginDialog dialog = new ServicePrincipalLoginDialog(project);
+        if (dialog.showAndGet()) {
+            AuthConfiguration data = dialog.getData();
+            authMethodDetailsResult = doServicePrincipalLoginInternal(data);
+            if (StringUtils.isNotBlank(data.getKey())) {
+                secureStore.savePassword("account", data.getClient(), data.getKey());
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/ui/SignInWindow.java:198]: verify combine key for secure store

Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/ui/SignInWindow.java:198]: Add logic to clear when logout

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/ui/SignInWindow.java [hunk 3]
```diff
@@ -268,7 +256,7 @@ private void checkAccountAvailability() {
                     oauthLoginRadioButton.setEnabled(false);
                     labelOAuthLogin.setEnabled(false);
                 }
-
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/ui/SignInWindow.java:271]: Move downgrade

Comment (andxu) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/ui/SignInWindow.java]: resolved in https://github.com/microsoft/azure-tools-for-java/pull/5265

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/ui/SignInWindow.java (hunk 1 above)
Comment (andxu) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/ui/SignInWindow.java]: resolved in https://github.com/microsoft/azure-tools-for-java/pull/5265

**Pull request merged**